infra/modules/database/role-manager.tf

```diff
@@ -29,7 +29,7 @@ resource "aws_lambda_function" "role_manager" {
       DB_PORT                = aws_rds_cluster.db.port
       DB_USER                = local.master_username
       DB_NAME                = aws_rds_cluster.db.database_name
-      DB_PASSWORD_PARAM_NAME = aws_ssm_parameter.random_db_password.name
+      DB_PASSWORD_PARAM_NAME = "/aws/reference/secretsmanager/${data.aws_secretsmanager_secret.db_pass.name}"
```

Comment:
I did some experimenting with this. Unfortunately this would require some additional IAM changes as well as I believe networking to enable access to the endpoint. Additionally, this is what was requested in the story